stable/logspout/templates/daemonset.yaml

```diff
@@ -0,0 +1,43 @@
+apiVersion: extensions/v1beta1
+kind: DaemonSet
+metadata:
+  name: {{ template "fullname" . }}
+  labels:
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}"
+    app: {{ template "fullname" . }}
+    heritage: "{{ .Release.Service }}"
+    component: logrouter
```

Comment:
The `component` doesn't seem to make much sense, since the chart doesn't have multiple components.